From fe94e6d3dd07ec9fa5ea40f2de44c6b29d520efd Mon Sep 17 00:00:00 2001
From: Fabien Parent <fparent@baylibre.com>
Date: Wed, 1 Dec 2021 19:52:18 +0100
Subject: [PATCH] Makefile: fix comparison operator

Fix the comparison operator in order for the installation of the plugins
to work correctly.

Signed-off-by: Fabien Parent <fparent@baylibre.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b3f16aa3478a..9359d9596495 100644
--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,7 @@ prepare-for-rootfs: examples
 			cp -p $$example/host/optee_example_$$example $(OUTPUT_DIR)/ca/; \
 		fi; \
 		cp -pr $$example/ta/*.ta $(OUTPUT_DIR)/ta/; \
-		if [ $$example == plugins ]; then \
+		if [ $$example = plugins ]; then \
 			cp -p plugins/syslog/*.plugin $(OUTPUT_DIR)/plugins/; \
 		fi; \
 	done
-- 
2.34.0

